common/council_tasks/00_steward_tasks.txt

```diff
@@ -584,9 +584,8 @@ task_promote_culture = {
 				desc = STEWARD_PROMOTE_CULTURE_COMMUNAL_IDENTITY_BONUS_MODIFIER
 			}
 		}
-
-		# Warcraft
-		# if = {
```

Comment:
Why?

Reply:
Why what exactly?

Reply:
Why you removed # Warcraft and broke indentation.